scripts/auto-rebase/rebase.sh

```diff
@@ -1226,10 +1229,37 @@ rebase_lvms_to() {
     rm -rf "${STAGING_DIR}"
 }
 
+to_just_images() {
+    local release_image_amd64=$1
+    local release_image_arm64=$2
+
+    download_release "${release_image_amd64}" "${release_image_arm64}"
```

Comment:
As a future enhancement, we could avoid cloning all of the git repos in this mode because we really just need the image list. Don't worry about doing that in this PR, we can optimize later.